apps/email/js/sync.js

```diff
@@ -77,22 +75,23 @@ define(function(require) {
           });
         }
 
-        title = title || mozL10n.get('notification-no-subject');
-
-        var notification = new Notification(title, notificationOptions);
-
-        // If the app is open, but in the background, when the notification
-        // comes in, then we do not get notifived via our mozSetMessageHandler
-        // that is set elsewhere. Instead need to listen to click event
-        // and synthesize an "event" ourselves.
-        notification.onclick = function() {
-          evt.emit('notification', {
-            clicked: true,
-            imageURL: iconUrl,
-            tag: notificationId,
-            data: data
-          });
-        };
+        var titleL10n = title;
```

Comment:
You can't just ignore the `notification-no-subject` scenario.
You need to do sth like:

```javascript
var titleL10n = title ? { raw: title } : 'notification-no-subject';
```